### PR TITLE
feat: update assignReviewer docs with policy

### PR DESCRIPTION
**Pull request by shay2025**
## Description

This pull request updates the documentation for `$assignReviewer` with the parameter `policy` that is implemented in the pull request https://github.com/reviewpad/reviewpad/pull/382

## Checklist

- [ ] I have performed a self-review of my code

## Code review and merge strategy (ship/show/ask) 

 
 
- [x] Ask: this pull request requires a code review before merge

### CODE REVIEW

**Comment by shay2025** [comment]
This pull request needs to be merged before https://github.com/reviewpad/reviewpad/pull/382

File: docs/guides/built-ins/actions/assignReviewer/index.mdx [hunk 1]
```diff
@@ -5,21 +5,27 @@ import Example from "!!raw-loader!./example.yml";
 
 Assigns a defined amount of reviewers to the pull request from the provided list of reviewers.
 
+The reviewers can be assigned following a policy of assignment that can **only** be one of the following:
+- `random`: the reviewers are chosen randomly;
+- `round-robin`: the reviewers are chosen following the `round-robin` algorithm;
+- `reviewpad`: the reviewers are chosen according to the number of pull requests they have already assigned to them - the reviewers that have less pull requests to review will be more likely to be assigned.
+
```
Comment (ferreiratiago) [inline docs/guides/built-ins/actions/assignReviewer/index.mdx:12]:
```suggestion
Reviewers can be assigned following one of the described policies:
- `random`: the reviewers are chosen in a random manner;
- `round-robin`: the reviewers are chosen following the `round-robin` algorithm;
- `reviewpad`: the reviewers are chosen based on the total number of pull requests assigned to them. The smaller the number of pull requests assigned to a reviewer, the greater the probability of being picked.
```

File: docs/guides/built-ins/actions/assignReviewer/index.mdx [hunk 2]
```diff
@@ -5,21 +5,27 @@ import Example from "!!raw-loader!./example.yml";
...
 **Parameters**:
 
-| Parameter          | Type      | Description                                                             |
-| ------------------ | --------- | ----------------------------------------------------------------------- |
-| `reviewers`        | \[]string | The list of GitHub logins to select from.                               |
-| `total` (optional) | int       | The total of reviewers to assign. By default, assigns to all reviewers. |
+| Parameter           | Type      | Description                                                                                     |
+| ------------------- | --------- | ----------------------------------------------------------------------------------------------- |
+| `reviewers`         | \[]string | The list of GitHub logins to select from.                                                       |
+| `total` (optional)  | int       | The total of reviewers to assign. By default, assigns to all reviewers.                         |
```
Comment (ferreiratiago) [inline docs/guides/built-ins/actions/assignReviewer/index.mdx:22]: For the description:
```
The total number of reviewers to assign to. By default, it assigns to all reviewers.
```

File: docs/guides/built-ins/actions/assignReviewer/index.mdx [hunk 3]
```diff
@@ -5,21 +5,27 @@ import Example from "!!raw-loader!./example.yml";
...
+| `policy` (optional) | string    | The arrangement of choosing the reviewers to assign. By default, the policy is `reviewpad`. |
```
Comment (ferreiratiago) [inline docs/guides/built-ins/actions/assignReviewer/index.mdx:23]: For the description: 
```
The policy followed for reviewer assignment. By default, the policy is `reviewpad`.
```

**Review by ferreiratiago — CHANGES_REQUESTED** [review]
Can you please review the uses cases to make sure no updates are required?

**Pull request merged**